a03_Mahak1729.py

```diff
@@ -0,0 +1,93 @@
+#################################################################################
+# Author: MAHAK KUMAWAT
+# Username:MAHAK1729
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: TO BUILD COOL STUFF AND LEARN FUNCTIONS
+# Google Doc Link: https://docs.google.com/document/d/1XCSB1JgXj2vmlOMzZ0txzvQJ5EK_cw7_2M5RI1vVwaM/edit]\
+#################################################################################
+# Acknowledgements:
+# Used My own code for starter code from HW02
+#################################################################################
+import turtle
+
+
+def sml_circle(shape):
+    shape.color("#c5dfd6")  # Sets circle color to light blue gray
+    shape.goto(50, - 70)
+    shape.pendown()
+    shape.begin_fill()
+    shape.circle(9)  # Draws small circle
+    shape.end_fill()
+    shape.penup()
+
+
+def big_circle(shape):
+    shape.color('#BDBDBD')  # Sets circle color to medium silver gray
+    shape.goto(50, - 70)
+    shape.pendown()
+    shape.begin_fill()
+    shape.circle(18)  # Draws big circle
+    shape.end_fill()
+    shape.penup()
+
+
+def main():
+    wn = turtle.Screen()  # Initiates turtle window
+    turtle.colormode(255)  # Gives a color range to choose the rgb from / limits to
+    wn.bgcolor('#ffb06d')  # Sets background color to light orange
+
+    shape = turtle.Turtle()  # Initiates turtle shape
+    shape.width(2)
+    shape.color('#B0BEC5')  # Sets initial turtle color to light grayish blue
+    shape.begin_fill()  # Fills with color
+    for side in range(3):  # Makes triangle(s)
+        shape.forward(100)
+        shape.left(120)
+    shape.end_fill()
+    shape.penup()
+
+    shape.color('#CFD8DC')  # Sets triangle turtle color to light bluish gray
+    shape.pendown()
+    shape.width(2)
+    shape.begin_fill()
+    for side in range(2):
+        shape.forward(100)
+        shape.right(90)
+        shape.forward(140)
+        shape.right(90)
+    shape.end_fill()
+    shape.penup()
+
+    big_circle(shape)  # Calling function that helps us draw big circle
+
+    sml_circle(shape)  # Calling function that helps us draw small circle
+
+    shape.goto(63, -140)  # Drawing flame
+    shape.pendown()
+    shape.color("#FF9800")  # Sets flame color to bright orange
+    shape.fillcolor("#FF9800")
+    shape.width(4)
+    shape.right(90)
+    shape.left(50)
+    shape.forward(40)
+    shape.right(125)
+    shape.forward(30)
+    shape.left(50)
+    shape.forward(30)
+    shape.right(125)
+    shape.forward(30)
+    shape.left(50)
+    shape.forward(30)
+    shape.right(125)
+    shape.forward(45)
+    shape.penup()
```

Comment:
Another function!

Reply:
I made the proper changes in the latest commit now for all the code/errors in it.